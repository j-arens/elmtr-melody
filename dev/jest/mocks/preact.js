module.exports = {
    unmountComponent: jest.fn(),
};
