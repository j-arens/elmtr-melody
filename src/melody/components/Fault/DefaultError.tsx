import Icon from '@components/Icon';
import { h } from 'preact';
const s = require('./style.scss');

export default () => (
    <span>
        <Icon className={s.errorContainer__icon} name="error" />
        <p class="melody__body">Something went wrong!</p>
    </span>
);
