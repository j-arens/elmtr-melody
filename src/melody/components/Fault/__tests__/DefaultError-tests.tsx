import { h } from 'preact';
import { render } from 'preact-render-to-string';
import DefaultError from '../DefaultError';

describe('DefaultError', () => {
    it('renders', () => {
        const component = render(<DefaultError/>);
        expect(component).toMatchSnapshot();
    });
});
