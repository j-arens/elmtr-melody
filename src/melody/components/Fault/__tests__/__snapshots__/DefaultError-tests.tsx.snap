// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultError renders 1`] = `
<p class="ErrorMsg">Something went wrong! Please try refreshing the page, otherwise contact
  support.</p>
`;
